package.json

```diff
@@ -15,7 +15,8 @@
     "styleguide": "styleguidist server",
     "styleguide:build": "styleguidist build && node scripts/moveBuildAssets.js",
     "test:watch": "npm test -- --watch --bail",
-    "test": "jest --runInBand --silent && npm run test:lint"
+    "test": "jest --silent && npm run test:lint",
```

Comment:
Any reason for removing `--runInBand`? IIRC that was needed for `SVGIcon` specs.